app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,70 @@
+class MoviesController < ApplicationController
+  before_action :set_movie, only: %i[ show edit update destroy ]
+  before_action :authenticate_user!, except: [:show, :index]
+  before_action :set_categories
+  # GET /movies or /movies.json
+  def index
+    @movies = Movie.all
+  end
+
+  # GET /movies/1 or /movies/1.json
+  def show
+  end
+
+  # GET /movies/new
+  def new
+    @movie = current_user.movies.build
```

Comment:
Why not Movie.new?

Reply:
I wanted to connect the movie to an user that added it, to later check if he is an admin for example